source/options.css

```diff
@@ -22,6 +22,11 @@ html::after {
 	}
 }
 
+/* Use sensible font size in Chrome #7483 */
+body {
+	font-size: 100%;
```

Comment:
We probably also want a sensible line-height, at least `1.3`. It's too tight as is

Reply:
It defaults to `normal` (which is `1.2` [according to MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#normal)). GitHub's `body`, `.markdown-body` and [Tailwind's preflight](https://github.com/tailwindlabs/tailwindcss/blob/62de02a37946c48c5fa6cdb800173b54198ebb9b/packages/tailwindcss/preflight.css#L30) use `1.5`, I assume it's fine to follow suit.